logging/google/cloud/logging/entries.py

```diff
@@ -170,14 +181,18 @@ class ProtobufEntry(_BaseEntry):
     :type http_request: dict
     :param http_request: (optional) info about HTTP request associated with
                          the entry
+
+    :type resource :class:`google.cloud.logging.resource.Resource`
```

Comment:
1. Please add a `~` before `google`
1. You are missing a colon after `resource`